.github/workflows/prerelease_windows.yml

```diff
@@ -104,12 +104,12 @@ jobs:
           username: ${{ env.DOCKER_HUB_ID }}
           password: ${{ env.DOCKER_HUB_PASSWORD }}
       - name: Publish to S3 action
-        uses: newrelic/infrastructure-publish-action@v1.0.4
+        uses: newrelic/infrastructure-publish-action@v1
```

Comment:
do we use different publis-action version for windos/linux?

Reply:
it should be v1 in the end, I am waiting for PR to be reviewed in Publish action to merge it into v1

Reply:
finally we decided to pin action version to specific one